generators/client/templates/angular/package.json.ejs

```diff
@@ -175,7 +175,7 @@
     "clean-www": "rimraf <%= DIST_DIR %>app/{src,<%= BUILD_DIR %>}",
     <%_ if (protractorTests) { _%>
     "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
-    "postinstall": "node node_modules/protractor/bin/webdriver-manager update --gecko false",
+    "postinstall": "webdriver-manager update --gecko false --versions.chrome ${CHROMEDRIVER_VERSION:-latest}",
```

Comment:
does it work under windows ?
I remember some changes about this, related to windows and I want to be sure it will work. 

see https://github.com/jhipster/generator-jhipster/pull/9643/files

Reply:
I don't think it works. Changed the approach.